android-core/src/main/java/com/mparticle/MParticle.java

```diff
@@ -1269,18 +1269,18 @@ public static void removeListener(SdkListener listener) {
      * @see #logEvent(BaseEvent)
      */
     public enum EventType {
-        Unknown(0), Navigation(1), Location(2), Search(3), Transaction(4), UserContent(5), UserPreference(6), Social(7), Other(8), Media(9);
 
+        Unknown(0), Navigation(1), Location(2), Search(3), Transaction(4), UserContent(5), UserPreference(6),
```

Comment:
this would be an incorrect public api change that'd likely cause a lot of pain and bugs :-) please revert

this enum is the one for custom events. it isn't meant to be used for all event types. ios does it this way and it's been a bug and complaint for years - customers for example log regular custom events with incorrect event types.

create some other private enum if we need it internally? these enum values shouldn't be exposed to customers.